app/models/commit_monitor_branch.rb

```diff
@@ -39,6 +39,14 @@ def pr_number
     MiqToolsServices::MiniGit.pr_number(name) if pull_request?
   end
 
+  def github_pr_uri
+    unless pull_request?
+      raise ArgumentError, "Cannot retrieve pull request URI for non-pull request " \
```

Comment:
Maybe just return nil if it's not a pull request.  Not sure if an exception is necessary.